analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -56,6 +56,8 @@ class Analyzer(private val config: AnalyzerConfiguration) {
         curationProvider: PackageCurationProvider? = null,
         repositoryConfigurationFile: File? = null
     ): OrtResult {
+        require(absoluteProjectPath.isAbsolute)
```

Comment:
Instead of failing, can't we just make the file absolute?

Reply:
Of course we could. But my understand is that we decided not to do that again here as we already do it in the caller, hence the naming of the argument.

For users of the API instead of the CLI it might indeed be more convenient if we did the conversion here (again), but I was a bit undecided about that and did not want to touch the code in that regard for now.

Reply:
I think we should change that sometime.